ctapipe/io/toymodel.py

```diff
@@ -76,12 +79,14 @@ def datalevels(self):
         return (DataLevel.DL1_IMAGES,)
 
     @property
-    def observation_block(self):
-        return dict()
+    def scheduling_blocks(self) -> Dict[int, SchedulingBlockContainer]:
+        return {-1: SchedulingBlockContainer(sb_id=-1, producer_id="ctapipe toymodel")}
```

Comment:
-1 doesn't make sense since you now defined it as uint64. Just using the default will keep the invalid value, so maybe just do:

```
sb = SchedulingBlock(produce_id="ctapipe toymode")
return {sb.sb_id: sb}
```

?